qiskit_nature/second_q/circuit/library/ansatzes/succd.py

```diff
@@ -106,27 +107,42 @@ def include_singles(self, include_singles: Tuple[bool, bool]) -> None:
         """Sets whether to include single excitations."""
         self._include_singles = include_singles
 
+    @property
+    def mirror(self) -> bool:
+        """Whether use SUCC_full"""
+        return self._mirror
+
+    @mirror.setter
+    def mirror(self, mirror: bool) -> None:
+        """Sets whether to use SUCC_full."""
+        self._mirror = mirror
+
+    def _check_length_excitations_list(self, operators, excitation_list):
+        self._excitation_list = self.generate_excitations(
+            self.num_spin_orbitals,
+            self.num_particles,
+        )
+        self.operators = operators
+
     def generate_excitations(
         self, num_spin_orbitals: int, num_particles: Tuple[int, int]
     ) -> List[Tuple[Tuple[int, ...], Tuple[int, ...]]]:
         """Generates the excitations for the SUCCD Ansatz.
-
```

Comment:
Please revert the changes to this docstring (in general, please do not change unrelated docstrings)